Source/DCSFlightpanels/PanelUserControls/PreProgrammed/Cdu737UserControlA10C.xaml.cs

```diff
@@ -0,0 +1,124 @@
+﻿namespace DCSFlightpanels.PanelUserControls.PreProgrammed
+{
+    using System.Windows;
+    using System.Windows.Controls;
+    using NonVisuals;
+    using NonVisuals.EventArgs;
+    using NonVisuals.Interfaces;
+
+    using NonVisuals.CockpitMaster.Preprogrammed;
+    using DCSFlightpanels.Interfaces;
+    using System;
+
+    /// <summary>
+    /// Logique d'interaction pour Cdu737UserControlA10C.xaml
```

Comment:
```suggestion
    /// Interaction logic for Cdu737UserControlA10C.xaml
```
Ha! un Visual Studio est en Français ^_^